src/lib.rs

```diff
@@ -71,6 +71,29 @@ pub struct Args {
     /// Enables more verbose output.
     #[structopt(long, short)]
     pub verbose: bool,
+
+    /// Specify the VCS used to initialize the generated template.
+    #[structopt(long, default_value = "git")]
```

Comment:
I've understood the original issue the way that you would not provide a `--vcs git` option but instead use `--git none` option.

Reply:
But I'm not 100% sure about it.

Reply:
hmmm, this is how I understood it - besides, `--git` is already taken for specifying the git repo for the template :-/

I selected to implement it this way as it also reminds of how it's done for `cargo` itself!

```shell
cargo-init 
Create a new cargo package in an existing directory

USAGE:
    cargo init [OPTIONS] [--] [path]

OPTIONS:
    -q, --quiet                  No output printed to stdout
        --registry <REGISTRY>    Registry to use
        --vcs <VCS>              Initialize a new repository for the given version control system (git, hg, pijul, or
                                 fossil) or do not initialize any version control at all (none), overriding a global
                                 configuration. [possible values: git, hg, pijul, fossil, none]
```

Reply:
probably you are right, consistency with cargo-init is what most users would naturally expect